Assets/Scripts/ThirdPersonPlayer.cs

```diff
@@ -106,6 +127,27 @@ private void OnDisable()
     {
         inputAction.Player.Disable();
     }
+    
```

Comment:
These subject-functions all do pretty much the same thing - toggle movement. Use a single toggle movement method.

Reply:
I did not write this code so I am not sure